stellargraph/data/explorer.py

```diff
@@ -256,36 +256,47 @@ def naive_weighted_choices(rs, weights):
 
 
 class BiasedRandomWalk(GraphWalk):
+
     """
     Performs biased second order random walks (like those used in Node2Vec algorithm
     https://snap.stanford.edu/node2vec/) controlled by the values of two parameters p and q.
+        
+    Args:
+        graph: The Stellargraph to perform uniform random walk on
+        graph_schema: The schema of the given graph
+        p: <float> Defines probability, 1/p, of returning to source node
```

Comment:
I'm not sure about providing p, q at `__init__` rather than at `run` the way it's done here:
- it's better if all parameters of the walk (p, q, n, length, etc.) are passed in one place: either at `__init__` or at `run`. Here p and q are passed to `__init__`, and n and length are passed to `run`. Might be confusing.
- our initial choice was to only pass the graph to `__init__`, so that a user could reuse the same walker, defined on a graph, by running it with different parameters. 
- we could switch to an alternative way of passing all the parameters to `__init__` instead, but then users will have to create multiple walker instances if they wanted to do something like parameter search for their problem, one walker instance per parameter set.
@adocherty @PantelisElinas do you have an opinion on this?